### PR TITLE
Update of autogenerated contribution information for round-1

### PR DESCRIPTION
**Pull request by eileen-kuehn**
This PR adds the current format of autogenerated contributions with the purpose to validate the formatting with the finalised PDF-generation workflow.

### CODE REVIEW

File: _events/software-demos/event-011.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ category: software-demos
 language: English
 prerequisites: "Basic familiarity with Python and the command line"
 license: Open source
-instructions: "<a href='https://www.sphinx-doc.org/en/master/usage/installation.html#installation-from-pypi'>https://sphinx-doc.org</a>"
+instructions: "https://www.sphinx-doc.org/en/master/usage/installation.html#installation-from-pypi"
```
Comment (Chilipp) [inline _events/software-demos/event-011.md:18]:
```suggestion
instructions: "<a href='https://www.sphinx-doc.org/en/master/usage/installation.html#installation-from-pypi'>https://sphinx-doc.org</a>"
```

File: _events/talks/event-009.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ language: English
 prerequisites: "General familiarity with issues from software development including concepts such as testing and automation as well as project management with limitations in academia are advantageous."
 date: 2020-07-17
 ---
-The challenges facing research software development are manifold and have long been a major topic at RSE conferences.
+The challenges facing research software development are manifold and have long been a major topic at RSE conferences.
```
Comment (Chilipp) [inline _events/talks/event-009.md:19]:
```suggestion
The challenges facing research software development are manifold and have long been a major topic at RSE conferences.
```
remove whitespace at the end of paragraph

**Comment by Chilipp** [comment]
one sec, the PDF generation did not trigger, I'll fix this quickly

**Comment by eileen-kuehn** [comment]
Thanks! I created tickets to automatically replace links and strip whitespace in the other repo.

**Comment by Chilipp** [comment]
alright, looks good to me, thanks @eileen-kuehn :smiley: Feel free to merge this if you are happy with it, too

**Pull request merged**